Makefile

```diff
@@ -98,6 +105,7 @@ binary: build ## build the linux binaries
 	$(DOCKER_RUN_DOCKER) hack/make.sh binary
```

Comment:
@tiborvass just to clarify, is it the case that setting the environment variable for the CLI path, and then running something like `make BIND_DIR=. binary shell` will set up a test `dind` container?